app/controllers/requests_controller.rb

```diff
@@ -100,13 +101,22 @@ def request_accept_button
 
   private
 
+  def save_sudo_rights(request)
+    sudo_users_for_request = request.users_assigned_to_requests.select { |uatq| request_params[:sudo_user_ids].include?(uatq.user_id.to_s) }
```

Comment:
what is uatq? if its user assign to request then this should be called userToRequest connection, assignedRequest or assoctiation like in the next line

Reply:
maby write the select block in multiple lines (this line seems way to long)